includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -2443,6 +2443,58 @@ public function schedule_order_tracking( $order_id, $order = null ) {
 		}
 	}
 
+	/**
+	 * Create a payment intent without confirming the intent.
+	 *
+	 * @param WC_Order $order - Order based on which to create intent.
+	 * @param array    $payment_methods - A list of allowed payment methods. Eg. card, card_present.
+	 * @param ?string  $capture_method - Controls when the funds will be captured from the customer's account ("automatic" or "manual").
+	 * It must be "manual" for in-person (terminal) payments.
+	 *
+	 * @return array An array containing the status (succeeded/failed), id (intent ID), message (error message if any), and http code.
+	 *
+	 * @throws Exception - When an error occurs in intent creation.
+	 */
+	public function create_intent( WC_Order $order, array $payment_methods, string $capture_method ) {
+		$amount           = $order->get_total();
+		$currency         = strtolower( $order->get_currency() );
+		$converted_amount = WC_Payments_Utils::prepare_amount( $amount, $currency );
```

Comment:
since the amount is used only here, you could use `$order->get_total()` and skip the `$amount` declaration

Reply:
The reason `$amount` exists is to annotate my code better without comments 😄 This way, it's instantly understandable by a new developer what the first argument of `prepare_amount` is. It also makes the code a bit cleaner.

Reply:
> Anyways, in case of prepare_amount with its only two arguments and one being self-explanatory, I think you can safely omit $account assignment.

Owing to the consensus, I've made the change 😄

Reply:
> I also find it outrageous that WC_Payments_Utils::prepare_amount doesn't do case conversion if it needs them and requires caller to take care of it. 👴

Indeed +1